source/server/admin/server_info_handler.cc

```diff
@@ -2,9 +2,10 @@
 
 #include "envoy/admin/v3/memory.pb.h"
 
+#include "source/common/http/headers.h"
```

Comment:
why did this header get added?  may have been VS Code automatically adding it

Reply:
I think it was being pulled in as a transitive dependency before. Without this header, the build fails:
```
source/server/admin/server_info_handler.cc: In member function 'Envoy::Http::Code Envoy::Server::ServerInfoHandler::handlerMemory(Envoy::Http::ResponseHeaderMap&, Envoy::Buffer::Instance&, Envoy::Server::AdminStream&)':
source/server/admin/server_info_handler.cc:45:50: error: 'Envoy::Http::Headers' has not been declared
   45 |   response_headers.setReferenceContentType(Http::Headers::get().ContentTypeValues.Json);
```